src/main/java/Methods.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+public class Methods {
+    public static void printGreetings() {
+        String logo = " ____        _        \n"
```

Comment:
You might want to consider declaring these variables as constants instead as they are they do not need to change.